src/libs/elektra/kdb.c

```diff
@@ -1597,52 +1593,51 @@ static bool runGetPhase (KeySet * backends, Key * parentKey, uint16_t phase)
+ * @pre The @p parentKey Key must be a valid Key, i.e., constructed with keyNew().
+ * @pre The @p parentKey Key must not have read-only name, value or metadata.
+ * @pre The @p parentKey Key must use the `spec:/`, `dir:/`, `user:/`, `system:/`,
+ * 	`default:/`, `proc:/` or cascading namespace.
  *
  * If you pass `NULL` or a key with read-only metadata as @p parentKey,
  * kdbGet() will fail immediately without doing anything.
  * If you pass another invalid @p parentKey, or `NULL` as @p ks or @p handle,
  * kdbGet() will set an error on @p parentKey and then return immediately.
  *
- * **Attention**: If you pass a non-NULL @p parentKey with writable metadata, kdbGet() will **always** remove
- *                any existing errors and warnings from @p parentKey.
+ * @note If you pass a non-NULL @p parentKey with writable metadata, kdbGet() will **always** remove
+ *       any existing errors and warnings from @p parentKey.
  *
- * **Important**: The KDB @p handle DOES NOT store a copy of the retrieved @p ks internally.
- *                This means if you later call kdbSet() with the same @p handle you must make sure to
- *                pass all keys from @p ks, which you do not want to remove.
+ * @warning The KDB @p handle DOES NOT store a copy of the retrieved @p ks internally.
+ *          This means if you later call kdbSet() with the same @p handle you must make sure to
+ *          pass all keys from @p ks, which you do not want to remove.
  *
  * @par Loadable Namespaces
  *
  * Not all namespace can be loaded.
- * `spec:/`, `dir:/`, `user:/` and `system:/` are loadable.
- * `proc:/` keys can be loaded but will never be persisted or cached.
- * `default:/` keys can be returned by kdbGet() but they will always stem from a specification in `spec:/` keys
- * If @p ks contains a key with any other namespace, an error will be returned.
- *
- * If @p parentKey is in any other namespace, an eror will be returned.
  *
- * **Note**: In general it is recommended to use a @p parentKey in the cascading namespace.
+ * - `spec:/`, `dir:/`, `user:/` and `system:/` can be loaded via kdbGet().
+ * - `proc:/` keys can be loaded via kdbGet(), but are not persisted or cached.
+ * - `default:/` keys can be inserted by kdbGet() but they will always stem from a specification in `spec:/` keys
+ * - If @p ks contains a key with any other namespace, an error will be returned.
  *
  * @par Parent Key
  *
  * The @p parentKey defines which parts of @p ks will be loaded.
  * Everything that is at or below @p parentKey wil be loaded together with any key
  * that shares a backend with such a key. Backends are always loaded as an atomic unit.
  *
- * **Note**: If @p parentKey is in the cascading namespace, keys of all loadable
- *           namspaces (see above) will be loaded.
+ * @note If @p parentKey is in the cascading namespace, keys of all loadable
+ *       namspaces (see above) will be loaded. This is generally the recommended approach.
```

Comment:
- Please use newlines as described in documentation guidelines for new sentences.
- Please add misspellings of namespaces to scripts/sed
- What is loadable? Please use consistent terminology (e.g. persistent)

```suggestion
 * @note If @p parentKey is in the cascading namespace, keys of all loadable namespaces (see above) will be loaded. 
 *       This is generally the recommended approach.
```

Reply:
> Please use newlines as described in documentation guidelines for new sentences.

I generally agree that every sentence should be on a separate line in Markdown files and other purely "docs text" files. However, in code files many (if not most) people will not have line wrapping enabled (for very good reasons), so putting each sentence on separate line in doxygen comments just makes it needlessly hard to read in the file. It also makes the file kind of ugly. All the C line will be automatically wrapped and reformatted at a certain length via `clang-format`. But doxygen comments will stick out and may go far beyond that limit.

In this specific case, it is probably still an acceptable line length, if we keep the sentence in one line, so accepted the change.

> What is loadable? Please use consistent terminology (e.g. persistent)

Do you need to say "See 16 lines above" and update it all the time???? The sentence already says "see above" and there is literally a section called "Loadable namespaces" above. Sometimes I truly think you purposefully ignore things.